tests/sanity/ignore-2.12.txt

```diff
@@ -0,0 +1,12 @@
+plugins/modules/digital_ocean_spaces.py compile-2.6!skip
+plugins/modules/digital_ocean_spaces_info.py compile-2.6!skip
+plugins/modules/digital_ocean_spaces.py compile-2.7!skip
+plugins/modules/digital_ocean_spaces_info.py compile-2.7!skip
+plugins/modules/digital_ocean_spaces.py compile-3.5!skip
+plugins/modules/digital_ocean_spaces_info.py compile-3.5!skip
```

Comment:
It's OK not to support 3.5 but it could be useful to specify the minimum required in the README (>=3.6?)

Reply:
A humble contribution: https://github.com/ansible-collections/community.digitalocean/pull/257